engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -1197,6 +1368,11 @@ namespace dmGameSystem
         }
         else if (IsReferencingProperty(SPRITE_PROP_SIZE, set_property))
         {
+            if (component->m_Resource->m_DDF->m_SizeMode == dmGameSystemDDF::SpriteDesc::SIZE_MODE_AUTO)
+            {
+                return dmGameObject::PROPERTY_RESULT_UNSUPPORTED_OPERATION;
+            }
```

Comment:
We disallow setting the property if the mode is auto since we get the size from the animation in that case.